### ISSUE TITLE
Not linking with the framework OpenGLES (referenced by a module reference in SkiaSharp.Views.iOS.dll) 

### ISSUE THREAD

**Issue by gktval** [mono/SkiaSharp#2414]
I am building SkiaSharp in a .Net Maui project on Mac but am receiving this error

ILLINK : warning MT0182: Not linking with the framework OpenGLES (referenced by a module reference in SkiaSharp.Views.iOS.dll) because it's not available on the current platform (MacCatalyst).

I have tried adding SkiaSharp.View.Maui.Controls to the main project but it did not help.

I also created a new, blank Maui project and added a SKCanvasView to the main page, it did not give me an error like my main project, but it did give me a warning with the same message.

**Comment by gktval** [comment]
No, the warning remains there. It seems to disappear and reappear frequently after rebuilding the solution. I'm not sure the rhyme or reason to it. But the app seems to work even with the warning.

**Comment by danies8** [comment]
I have reproduced the warning outside of the usage of Telerik controls. Sample maui project just added the SkiaSharp.Views.Maui.Controls and Compatibilityreference and called .UseSkiaSharp inside the MauiProgram.cs file.

**Comment by didiyordanova** [comment]
@gktval @danies8 @mattleibow @drossoft

I cannot reproduce the warning using SkiaSharp 3.116.1 version. Could you please share what is the exact SkiaSharp version you use in the .NET MAUI app?
